### ISSUE TITLE
Installation procedure for Centos 8

### ISSUE THREAD

**Issue by pnax** [zonemaster/zonemaster-backend#680]
### Starman installation fails
While following the [Installation process on Centos 8](https://github.com/zonemaster/zonemaster-backend/blob/master/docs/Installation.md#31-install-zonemasterbackend-and-related-dependencies-centos), there are the following statements :

*  Only for CentOS 7 *

```sh
sudo yum install perl-Plack-Test
```

Install dependencies not available from binary packages:

```sh
sudo cpanm Daemon::Control Starman
```

So I skipped the `perl-Plack-Test` installation and went straigth to the last command.

But installing `Starman` fails. It seems that installing `perl-Plack-Test` resolves the problem.

edit: it looks like this was already discussed in https://github.com/zonemaster/zonemaster-backend/pull/663

### Make distinction between Centos 7 and Centos 8 more obvious
Sections titles [`3.2.2.1`](https://github.com/zonemaster/zonemaster-backend/blob/master/docs/Installation.md#3221-postgresql-installation-instructions-for-centos7) and [`3.2.2.2`](https://github.com/zonemaster/zonemaster-backend/blob/master/docs/Installation.md#3222-postgresql-installation-instructions-for-centos8) are rendered very small.
It would be nice to make them bigger or to make it more obvious that they each refer to different versions of Centos.

**Comment by matsduf** [comment]
If `Starman` requires `Plack::Test` and is installed from CPAN I am surprised that is does not pull in `Plack::Test` from CPAN.

**Comment by mattias-p** [comment]
@matsduf Makefile.PL lists the Plack::Builder module which is provided by the Plack CPAN distribution. The Plack::Test module is also provided by the Plack CPAN distribution. On CentOS however, if you only install perl-Plack you get Plack::Builder but not Plack::Test.

I have a recollection of asking the maintainer of Plack about this at some point and he got upset about the state of affairs on CentOS. Sadly I can't find any traces of the conversation. Maybe it wasn't Plack at all but some other Perl module.

In any case CentOS is doing something weird with their packaging here and we have to work around it by including perl-Plack-Test in the installation instruction.

**Comment by matsduf** [comment]
@pnax, has this been fixed or is it still relevant?

**Comment by pnax** [comment]
This has been fixed in #696
